commit aad1fb44a3ea8de9b079de567bcdd837f8f978e3
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Apr 29 19:16:15 2019 +0100

    Revert "staging: bcm2835-audio: Enable compile test"
    
    commit 4eae66777a262ac9707980ea0cfe902afadfb577 upstream.
    
    This reverts commit 02d205a57c4c943fc2a5b1ac7c912ce01944f700.

diff --git a/drivers/staging/vc04_services/bcm2835-audio/Kconfig b/drivers/staging/vc04_services/bcm2835-audio/Kconfig
index 62c1c8ba4ad4..9f536533c257 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/Kconfig
+++ b/drivers/staging/vc04_services/bcm2835-audio/Kconfig
@@ -1,6 +1,6 @@
 config SND_BCM2835
         tristate "BCM2835 Audio"
-        depends on (ARCH_BCM2835 || COMPILE_TEST) && SND
+        depends on ARCH_BCM2835 && SND
         select SND_PCM
         select BCM2835_VCHIQ
         help
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1439_aad1fb44a3ea8de9b079de567bcdd837f8f978e3.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1439_aad1fb44a3ea8de9b079de567bcdd837f8f978e3.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
